openfisca_france_dotations_locales/variables/dotation_solidarite_rurale.py

```diff
@@ -169,6 +185,56 @@ def formula(commune, period, parameters):
         return preeligible * not_(dsr_exclue_fraction_bourg_centre_type_1)
 
 
+class dsr_eligible_fraction_bourg_centre_type_2(Variable):
+    value_type = bool
+    entity = Commune
+    definition_period = YEAR
+    label = "eligibilité à la fraction bourg-centre de la DSR pour les \
+        communes de taille inférieure au seuil de 10000 habitants"
+    reference = [
+        'Code général des collectivités territoriales - Article L2334-21',
+        'https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000036433099&cidTexte=LEGITEXT000006070633',
+        "http://www.dotations-dgcl.interieur.gouv.fr/consultation/documentAffichage.php?id=94"
+        ]
+    documentation = '''
+        Bénéficient également de cette fraction les chefs-lieux d'arrondissement
+        au 31 décembre 2014, dont la population est comprise entre 10 000 et
+        20 000 habitants, qui n'entrent pas dans les cas prévus aux 1° et 4° ci-dessus.
+    '''
+
+    def formula(commune, period, parameters):
+        population_dgf_plafonnee = commune("population_dgf_plafonnee", period)
+        outre_mer = commune('outre_mer', period)
+        chef_lieu_arrondissement = commune("chef_lieu_arrondissement", period)
+        dsr_exclue_fraction_bourg_centre_type_2 = commune('dsr_exclue_fraction_bourg_centre_type_2', period)
+
+        taille_max_commune = parameters(period).dotation_solidarite_rurale.seuil_nombre_habitants
+        taille_max_chef_lieu_arrondissement = parameters(period).dotation_solidarite_rurale.bourg_centre.eligibilite.seuil_nombre_habitants_chef_lieu
+        taille_eligible = (population_dgf_plafonnee >= taille_max_commune) * (population_dgf_plafonnee < taille_max_chef_lieu_arrondissement)
```

Comment:
Les 20 000 ne sont-ils pas inclus d'après le texte ? 🤔 
```suggestion
        taille_eligible = (population_dgf_plafonnee >= taille_max_commune) * (population_dgf_plafonnee <= taille_max_chef_lieu_arrondissement)
```